### PR TITLE
Align CLI to spicepod format

### PR DESCRIPTION
**Pull request by mitchdevenport**
Builds on https://github.com/spiceai/spiceai/pull/634 to make further required changes to the CLI to support new spicepod format.

### CODE REVIEW

File: bin/spice/pkg/api/pod.go [hunk 1]
```diff
@@ -12,6 +12,6 @@ type Pod struct {
 }
 
 type Reference struct {
-	From      string `json:"from,omitempty" csv:"from" yaml:"from,omitempty"`
+	Import    string `json:"import,omitempty" csv:"import" yaml:"import,omitempty"`
```
Comment (lukekim) [inline bin/spice/pkg/api/pod.go:15]: Is this expected?

File: bin/spice/pkg/cli/cmd/dataset.go [hunk 2]
```diff
@@ -54,9 +54,16 @@ spice dataset configure
 		accelerateDatasetString = strings.TrimSuffix(accelerateDatasetString, "\n")
 		accelerateDataset := strings.ToLower(accelerateDatasetString) == "y"
 
+		params := map[string]string{}
+		if strings.Split(datasetLocation, "/")[0] == api.DATA_SOURCE_DREMIO {
+			// TODO: Allow user to specify own dremio instance. Needs UX design for how the command should handle.
+			params["url"] = "http://dremio-4mimamg7rdeve.eastus.cloudapp.azure.com:32010"
```
Comment (lukekim) [inline bin/spice/pkg/cli/cmd/dataset.go:60]: I'd called this `endpoint` if it includes schema and port.

File: bin/spice/pkg/api/pod.go (hunk 1 above)
Comment (lukekim) [inline bin/spice/pkg/api/pod.go:15]: Maybe we should call this `Ref`?

**Review by lukekim — APPROVED** [review]

File: bin/spice/pkg/api/pod.go (hunk 1 above)
Comment (mitchdevenport) [reply bin/spice/pkg/api/pod.go:15]: Yeah I like that

**Pull request merged**